src/js/sidebar/course-builder/editor.scss

```diff
@@ -0,0 +1,68 @@
+.edit-post-post-status {
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+
+	h2 {
+		order: -2;
+	}
+}
+
+.llms-launch-course-builder {
+	order: -1;
+	margin: 0 0 16px;
+}
+
+.llms-primary-background-color {
+	background-color: #466dd8 !important;
+	color: #fff !important;
+
+	&:hover,
+	&.clicked {
+		background-color: #2b55cb !important;
+	}
+}
+
+.llms-course-builder-panel {
+
+	&--close {
+
+		.components-panel__body-title {
+			margin-bottom: -16px !important;
+		}
+
+		.components-panel__arrow {
+			transform: rotate(180deg) translateY(50%) !important;
+		}
+
+		> div {
+			display: none;
+		}
+	}
+}
+
+#side-sortables #course_builder {
```

Comment:
we hide the classic metabox here right?
Might be worthwhile do the relevant changes in the code plugin in order to not register the metabox at all if the post is built with gutenberg/block editor?

Reply:
The new changes actually use the classic metabox still, it is just repositioned into the plugin sidebar with `subscribe`. 

Since the metabox contained a lot of PHP logic, it was easier to keep it as-is, instead of rebuilding in JS. 

I do need to check to make sure this doesn't hide the metabox in the Classic editor though!

Reply:
Oh that's nice.
But wait, the subscribe isn't used for the course builder launch button only?

Am I wrong or you're cloning the "classic metabox" nodes, and appending them to the new panel?
If so can't we just move them, and once finished remove their metabox wrapper too?

> I do need to check to make sure this doesn't hide the metabox in the Classic editor though!
At the moment it does hide the metabox in the classic editor, so yeah it needs a fix :D